cmd/core-command/res/configuration.yaml

```diff
@@ -13,11 +13,20 @@ Service:
   Host: localhost
   Port: 59882
   StartupMsg: "This is the Core Command Microservice"
+  SecurityOptions:
+    Mode: "zerotrust"
+    OpenZitiController: "openziti:1280"
+    OpenZitiServiceName: "edgex.core-command"
```

Comment:
Put these in the `all-services` section of core-common-config-bootstrapper
and remove them from each service's private config